### ISSUE TITLE
Files moved

### ISSUE THREAD

**Issue by SlimeDog** [plan-player-analytics/Plan#991]
Plan 4.8.0 final

Due to my heavy customization, I always unzip the JAR and check for differences.

The following files are now found under _assets/plan_. Until this release, they were found in the root directory:
- bungeeconfig.yml
- config.yml
- DefaultServerInfoFile.yml
- locale
- themes
- web

Other files remain in the root directory:
- bungee.yml
- Cert.keystore
- velocity-plugin.json

Just wondering...

**Comment by Rsl1122** [comment]
This is related to fix for issue #922

On Sponge files inside jar need to be accessed through their Asset API which requires the files to be under /assets/pluginid/ inside the jar.

The file accessing was refactored to fix wrong config appearing on Sponge sometimes.

**Comment by SlimeDog** [comment]
Thanks. No issue for me; my difference engine will find them. :)

**Comment by SlimeDog** [comment]
So, my customized files are not found. Do I need to recreate them in _assets/plan/_ ?

**Comment by Rsl1122** [comment]
Hmm, I might have messed something up with the fetching of the customizable files. I'll look into it when I get home.

**Comment by Rsl1122** [comment]
A tree structure of the customized files will help, but is not necessary.

On most distros `tree` prints tree structure of the current directory.

**Comment by SlimeDog** [comment]
Until now, the tree was a reflection of the files in the JAR that I wanted to customize:
```
$ ls -R1 plugins/Plan/web/
plugins/Plan/web/:
error.html
network.html
player.html
players.html
server.html

plugins/Plan/web/css:
main.css
style.css

plugins/Plan/web/js/charts:
diskGraph.js
lineGraph.js
pingGraph.js
playerGraph.js
resourceGraph.js
tpsGraph.js
worldGraph.js
```

**Comment by Rsl1122** [comment]
Looks like I haven't been focusing enough during the file refactoring.

The bug was stupid: It was attempting to find a file with the full path as name (eg web/js/charts/diskGraph.js) instead of going one part at a time.

I'll release a 4.8.1 DEV1 with a fix shortly.

**Comment by SlimeDog** [comment]
Thanks. No worries. That's why I have a test server.

**Comment by SlimeDog** [comment]
Confirmed fixed: Plan 4.8.0-b1
